commands/delete-key.js

```diff
@@ -20,6 +21,25 @@ async function deleteAccessKey(options) {
     console.log(`Deleting key = ${options.accessKey} on ${options.accountId}.`);
     const near = await connect(options);
     const account = await near.account(options.accountId);
+    let accessKeys = await account.getAccessKeys();
+    let fullAccessKeys = accessKeys.filter(accessKey => accessKey.access_key.permission === 'FullAccess');
+
+    if (fullAccessKeys.length === 1 && fullAccessKeys[0].public_key.includes(options.accessKey)){
+        const rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout
+        });
+
+        const question = (str) => new Promise(resolve => rl.question(str, resolve));
+        const answer = await question('WARN: you want to remove the last full access key and forgot access to this account [Y/n]? ');
+
+        if(['N', 'n', 'no', 'No', 'NO'].includes(answer)) {
```

Comment:
Thank you @TarasenkoFirelabs! This contribution is super important.
Since we want to ensure that the user will not delete his last key by accident, let's check for `YES`/`Yes`/`Y`/`y` and delete key `ONLY` if these answers are provided.
Also, with this approach we will not use `precess.exit(1)` which is not a good practice.

Reply:
solved